apps/desktop/src/platform/services/ssh-agent.service.ts

```diff
@@ -153,7 +153,10 @@ export class SshAgentService implements OnDestroy {
 
           if (isListRequest) {
             const sshCiphers = ciphers.filter(
-              (cipher) => cipher.type === CipherType.SshKey && !cipher.isDeleted,
+              (cipher) =>
+                cipher.type === CipherType.SshKey &&
+                !cipher.isDeleted &&
+                cipher.organizationId === null,
```

Comment:
I'd recommend doing a non-strict equals against null. 

```suggestion
                cipher.organizationId == null,
```

Reply:
Updated! Since we also used the strict equal a bit below in the same file, I replaced that too.